testing/junit/src/test/java/com/linecorp/armeria/testing/junit/server/mock/MockWebServiceExtensionTest.java

```diff
@@ -103,21 +104,15 @@ void normal() {
 
     @Test
     void delay() {
-        server.enqueue(MockResponse.builder(AggregatedHttpResponse.of(HttpStatus.OK))
-                                   .delay(Duration.ofMillis(200))
-                                   .build());
-        server.enqueue(MockResponse.builder(AggregatedHttpResponse.of(HttpStatus.OK))
-                                   .delayMillis(300)
-                                   .build());
+        server.enqueue(HttpResponse.delayed(HttpResponse.of(AggregatedHttpResponse.of(HttpStatus.OK)),
```

Comment:
What do you think about adding `AggregatedHttpResponse.delayed()` as well?

Reply:
Ugh, actually, `HttpResponse.delayed(AggregatedHttpResponse)`.

Reply:
Sure